duckduckgo/iOS
# Update Gemfile to use fastlane fork

Task/Issue URL: https://app.asana.com/0/1203301625297703/1203772337302927/f

**Description**:
Update to DDG's fork with a fix for https://github.com/fastlane/fastlane/issues/21003



**Steps to test this PR**:
We'll really be able to test it while releasing, but for now you can e.g. try `bundle exec fastlane adhoc` to verify that fastlane fork isn't broken.




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
